File: testprojects/csharp/benchmark/domain/Indirect/AlowedFrom/CallInstanceMethodIndirect_StaticMethodInstanceMethod.cs
namespace Domain.Indirect.AllowedFrom
{
	public class CallInstanceMethodIndirect_StaticMethodInstanceMethod
	{
		
		public void Case5()
		{
            string s = Domain.Indirect.Intermediate.BackgroundService.GetServiceOneviaStaticAttribute().GetName();
		}
	}
}

File: testprojects/java/accuracy/domain/indirect/allowedfrom/CallInstanceMethodIndirect_StaticMethodInstanceMethod.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class CallInstanceMethodIndirect_StaticMethodInstanceMethod {
	
	public CallInstanceMethodIndirect_StaticMethodInstanceMethod() {
	}

	// Indirect invocation via static method
	public void case5(){
		String s = BackgroundService.getServiceOneviaStaticAttribute().getName();
		System.out.println(s);
		
	}
}
